conf/modules/stereocam_stereocam2state.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="stereocam2state" dir="stereocam/stereocam2state">
+  <doc>
+    <description>This module sends the data retreived from an external stereocamera modules, to the state filter of the drone. This is done so that the guidance modules can use that information for couadcopter</description>
```

Comment:
First line of description should be the brief description "name" of the module (see https://wiki.paparazziuav.org/wiki/Modules#In_the_module_file) as it is used in the generated docs at http://docs.paparazziuav.org/latest/onboard_modules.html